netbout-client/pom.xml

```diff
@@ -180,7 +180,12 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in client
```

Comment:
@dmzaytsev this is what I'm saying - this puzzle is bigger than 1 line. I just trying to understand what's wrong except absence of ":30min"